Formula/victoriametrics.rb

```diff
@@ -0,0 +1,69 @@
+class Victoriametrics < Formula
+  desc "Cost-effective and scalable monitoring solution and time series database"
+  homepage "https://victoriametrics.com/"
+  url "https://github.com/VictoriaMetrics/VictoriaMetrics/archive/v1.88.1.tar.gz"
+  sha256 "92803ae61f927b7173e25d4537c429bf3054edfa2f719bafc8fb9fcd411cab87"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+  depends_on "make" => :build
+
+  def install
+    ENV.deparallelize
+
+    system "make", "victoria-metrics"
+    bin.install "bin/victoria-metrics"
+
+    (etc/"scrape.yml").write <<~EOS
+      global:
+        scrape_interval: 10s
+
+      scrape_configs:
+        - job_name: "victoriametrics"
+          static_configs:
+          - targets: ["127.0.0.1:8428"]
+    EOS
+    etc.install "scrape.yml"
+  end
+
+  service do
+    run [opt_bin/"victoriametrics_brew_services",
+      "--promscrape.config=#{etc}/scrape.yml",
+      "--storageDataPath=#{var}/victoriametrics-data",
+      "--retentionPeriod=12",
+      "--httpListenAddr=127.0.0.1:8428",
+      "--graphiteListenAddr=:2003",
+      "--opentsdbListenAddr=:4242",
+      "--influxListenAddr=:8089",
+      "--enableTCP6"]
```

Comment:
Aren't these the default?

Reply:
Not these flags are not default. But why they can't be configure thrue the brew install?

Reply:
They can, but why not use the defaults here? If the defaults are bad we should encourage upstream to set better ones

Reply:
Removed these lines.